Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.py

```diff
@@ -334,6 +589,13 @@ def main() -> None:
     token = params.get("token")
     check_cert = params.get("check_cert", False)
     proxy = params.get("proxy", False)
+    reliability = params.get('integrationReliability')
```

Comment:
you can do a small refactoring here after you use the type 15
(see my comment in the definition of `integrationReliability` in the YML fiile)

Reply:
done